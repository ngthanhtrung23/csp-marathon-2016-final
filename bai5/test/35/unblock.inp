1.2233
1.4555
004.67
.88967
AAA96B
.....B
