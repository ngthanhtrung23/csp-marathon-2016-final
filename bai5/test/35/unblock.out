35
2 L 1
3 L 1
8 L 1
4 D 1
5 L 2
6 U 1
7 U 2
9 U 1
B U 2
A R 3
4 D 2
0 R 1
8 R 1
1 D 4
0 L 1
2 L 1
8 L 1
4 U 2
A L 2
6 D 2
B D 2
7 D 2
5 R 2
4 U 2
0 R 1
8 R 1
1 U 3
8 L 1
A L 1
9 D 1
0 R 2
4 D 2
5 L 1
7 U 2
0 R 1

